share/store/file/file.go

```diff
@@ -22,11 +22,11 @@ type EdsFile interface {
 	// DataHash returns data hash of the file.
 	DataHash() share.DataHash
 	// Share returns share and corresponding proof for the given axis and share index in this axis.
-	Share(ctx context.Context, x, y int) (*share.ShareWithProof, error)
+	Share(ctx context.Context, x, y int) (*share.Sample, error)
 	// AxisHalf returns Shares for the first half of the axis of the given type and index.
 	AxisHalf(ctx context.Context, axisType rsmt2d.Axis, axisIdx int) (AxisHalf, error)
```

Comment:
random idea to alias rstm2d.Axis to a type defined in share pkg, but def optional